includes/Core/Authentication/Authentication.php

```diff
@@ -238,14 +233,24 @@ public function verification() {
 	/**
 	 * Gets the verification tag instance.
 	 *
+	 * @return Verification_Meta Verification tag instance.
 	 * @since 1.0.0
-	 *
-	 * @return Verification_Tag Verification tag instance.
 	 */
 	public function verification_tag() {
```

Comment:
Same here. I suggest introducing a new method `verification_meta()` (`@since n.e.x.t`) that does this, and then in the existing `verification_tag()`, use `_deprecated_function( __METHOD__, 'n.e.x.t', __CLASS__ . '::verification_meta()' )` and just wrap `verification_meta()` so that it still works. Also make sure to add a `@deprecated n.e.x.t` annotation to `verification_tag()`.

We haven't been strict about public-facing changes like this so far, and there's always a line to draw somewhere, but I think this would be a good place to deprecate.

Reply:
Done!

Reply:
Please keep the since entry above the deprecated entry, this is how WP core handles it too.

Reply:
Agh, my IDE keeps moving `since` to the bottom for some reason. Can we enforce ordering with phpcs?